### PR TITLE
Shard related code removed

### PR DESCRIPTION
**Pull request by motoras**
For details see [ticket](https://radixdlt.atlassian.net/browse/RPNV1-309)

### CODE REVIEW

File: radixdlt/src/main/java/org/radix/GenerateNodeKey.java [hunk 1]
```diff
@@ -17,45 +17,36 @@
```
Comment (clstrfsck) [inline radixdlt/src/main/java/org/radix/GenerateNodeKey.java]: This file should be deleted.  It was for a specific purpose to generate keys in specific shards for 1mtps project, and is no longer required.

**Review by clstrfsck — CHANGES_REQUESTED** [review]
Please remove the commented out GenerateNodeKey class, otherwise looks good.

File: radixdlt/src/main/java/org/radix/GenerateNodeKey.java (hunk 1 above)
Comment (motoras) [reply radixdlt/src/main/java/org/radix/GenerateNodeKey.java]: Done

**Review by talekhinezh — APPROVED** [review]

**Review by clstrfsck — APPROVED** [review]
LGTM

**Pull request merged**